```ShellSession
$ nix-build -A python2Packages.deskcon
...
Executing setuptoolsBuildPhase
Traceback (most recent call last):
File "nix_run_setup", line 8, in <module>
exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
IOError: [Errno 2] No such file or directory: 'setup.py'
error: build of '/nix/store/3635ygwkgh91qprwxpbh5dq1862ndh04-python2.7-deskcon-0.3.drv' on 'ssh-ng://bill.r' failed: error: builder for '/nix/store/3635ygwkgh91qprwxpbh5dq1862ndh04-python2.7-deskcon-0.3.drv' failed with exit code 1;
last 10 log lines:
> setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/unity/indicator-deskcon.svg
> patching sources
> configuring
> no configure script, doing nothing
> building
> Executing setuptoolsBuildPhase
> Traceback (most recent call last):
>   File "nix_run_setup", line 8, in <module>
>     exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
> IOError: [Errno 2] No such file or directory: 'setup.py'
For full logs, run 'nix log /nix/store/3635ygwkgh91qprwxpbh5dq1862ndh04-python2.7-deskcon-0.3.drv'.
error: builder for '/nix/store/3635ygwkgh91qprwxpbh5dq1862ndh04-python2.7-deskcon-0.3.drv' failed with exit code 1
```

--- a/pkgs/development/python-modules/deskcon/default.nix
+++ b/pkgs/development/python-modules/deskcon/default.nix
@@ -18,7 +18,8 @@ buildPythonPackage {
     sha256 ="0i1dd85ls6n14m9q7lkympms1w3x0pqyaxvalq82s4xnjdv585j3";
   };
 
-  phases = [ "unpackPhase" "installPhase" ];
+  dontBuild = true;
+  doCheck = false;
 
   pythonPath = [ pyopenssl pkgs.gtk3 ];
 

